releasetool/github.py

```diff
@@ -70,6 +91,37 @@ def __init__(self, token: str, use_proxy: bool = False) -> None:
             # To use the proxy, we need an api key for the magic github proxy.
             self.session.params = {"key": _find_devrel_api_key()}
 
+    def application_access_token(
+        self, app_id: str, installation: str, private_key_str: str
```

Comment:
I don't see the self parameter used in the body of this function.

Therefore, could we move it out of the class, and into a stand-alone function, and add a """doc string"""?  That will make it easier to write a test too.